src/Adapters/EventSource.Netstandard13/EventSourceTelemetryModule.cs

```diff
@@ -154,9 +154,9 @@ protected override void OnEventSourceCreated(EventSource eventSource)
         private void EnableAsNecessary(EventSource eventSource)
         {
             // Special case: enable TPL activity flow for better tracing of nested activities.
-            if (eventSource.Guid == EventSourceTelemetryModule.TplEventSourceGuid)
+            if (eventSource.Guid == TplActivities.TplEventSourceGuid)
             {
-                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords)EventSourceTelemetryModule.TaskFlowActivityIds);
+                this.EnableEvents(eventSource, EventLevel.LogAlways, (EventKeywords) TplActivities.TaskFlowActivityIdsKeyword);
```

Comment:
@SergeyKanzhelev  What is the nature of the problem? 

Enabling the TPL EventSource is definitively valuable because of the hierarchical activity ID benefit. It is a great help in figuring out how much time each step of a complicated activity took (say handling of a request that involves several async calls to dependencies)